dagshub/upload/wrapper.py

```diff
@@ -23,6 +23,18 @@
 
 
 def get_default_branch(src_url, owner, reponame, auth):
+    """
+    The get_default_branch function takes in a source URL, owner name and repo name.
+    It then uses the GitHub API to get the default branch of that repo.
+    
+    Args:
+        src_url[str]: Construct the url for the api call
+        owner[str]: Specify the owner of the repo
+        reponame[str]: Construct the url to request
+        auth[str]: Authenticate the request
+    Returns:
+        The default branch for a repository
+    """
     res = requests.get(urllib.parse.urljoin(src_url, REPO_INFO_URL.format(
```

Comment:
Looks good, but this function recently moved to `dagshub/common/helpers.py` and changed a bit.
It will cause a merge conflict so need to be moved as well.